extension-workspace-references.md

```diff
@@ -43,9 +102,9 @@ _Request_
  */
 interface WorkspaceReferencesParams {
     /**
-     * Metadata about the symbol that is being searched for.
+     * Known properties about the identity of the symbol (for example, the package identifier)
```

Comment:
Should you define the affect of this parameter on the behavior of `workspace/references`? E.g., "filters the list of returned references to those that reference a symbol that match the defined fields in `query`"